### ISSUE TITLE
Using pfx to serve on https

### ISSUE THREAD

**Issue by crowmw** [webpack-contrib/webpack-serve#101]
I migrating my application from webpack-dev-sever to webpack-serve
I can't run developer server on https using described options (https-pfx/https-pass)
On webpack-dev-server everything works well with the same configuration
I missed something?

* Operating System: Windows 10
* Node Version: 8.9.0
* NPM Version: 5.8.0
* webpack Version: 4.5.0
* webpack-serve Version: 0.3.1

##### CLI Command

```console
  $ webpack-serve --config ./webpack.config.js --https-pfx=./Config/localhost.certificate.pfx --https-pass=custompass123
```

### Expected Behavior

serve application on https server

### Actual Behavior

```console
  An error was thrown while starting webpack-serve.                                                                    
   Error: not enough data                                                                                            
    at Object.createSecureContext (_tls_common.js:160:19)                                                            
    at Server (_tls_wrap.js:803:25)                                                                                  
    at new Server (https.js:54:14)                                                                                   
    at Object.createServer (https.js:76:10)                                                                          
    at module.exports (F:\PA3\ContactCenter\PALite\SccWeb\node_modules\webpack-serve\lib\server.js:63:22)            
    at getOptions.then (F:\PA3\ContactCenter\PALite\SccWeb\node_modules\webpack-serve\index.js:63:40)                
    at <anonymous>                                                                                                   
    at process._tickCallback (internal/process/next_tick.js:188:7)                                                   
    at Function.Module.runMain (module.js:686:11)                                                                    
    at startup (bootstrap_node.js:187:16)           
```

**Comment by shellscape** [comment]
Thanks for the issue :beer: Happy to review a PR to resolve this.

**Comment by Seebiscuit** [comment]
Pretty much the same setup and received the same Error/StrackTrace. 

Any updates on this?

**Comment by shellscape** [comment]
@Seebiscuit when there's an update you'll see it listed in the issue thread.

**Comment by Seebiscuit** [comment]
Suppose I want to contribute. I've never looked at the wp-serve code. Is this bug approachable?

**Comment by Seebiscuit** [comment]
So, I found this, which may very well be the fix you're looking for: [P12 certificate “Not enough data” error
](https://stackoverflow.com/a/37300058/2112866). Thoughts?

**Comment by shellscape** [comment]
Looks like a good lead

**Comment by shellscape** [comment]
Fixed in #142
